AdminServer/appscale/admin/version.schema.json

```diff
@@ -0,0 +1,173 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "AppEngine application version",
+    "description": "An AppEngine application version - apps.services.versions.Version",
+    "definitions": {
+        "duration": {
+            "type": "string",
+            "pattern": "^[+-]?[0-9]{1,12}(?:.[+-]?[0-9]{1,9})?s$"
+        }
+    },
+    "type": "object",
+    "properties": {
+        "automaticScaling": {
```

Comment:
Is there a way to mark these keys as exclusive so that a client cannot define both automaticScaling and manualScaling?